generated/provider_dependencies.json

```diff
@@ -221,7 +221,9 @@
     ],
     "devel-deps": [],
     "plugins": [],
-    "cross-providers-deps": [],
+    "cross-providers-deps": [
```

Comment:
I am assuming that the `cross-providers-deps` is for optional dependencies. If so, do we need to add `apache-kafka` provider to the `cross-providers-deps` of `google` provider?

Reply:
Ohh, I see. I think we can some feedback on public PR then.